e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -53,6 +55,74 @@ describe("banner", () => {
         ).should("not.exist");
       });
   });
+
+  describe("embeddings", () => {
+    const QUESTION_ID = 1;
+    it("should not render for public links", () => {
+      cy.request("POST", `/api/card/${QUESTION_ID}/public_link`, {}).then(
+        ({ body: { uuid } }) => {
+          cy.visit(`/public/question/${uuid}`);
+        },
+      );
+
+      cy.findAllByRole("banner")
```

Comment:
This will likely always pass because the check will happen before anything is loaded. Could you assert that something is present on the page first? Same for other tests

Reply:
Fixed